src/main.ts

```diff
@@ -58,7 +67,18 @@ async function run() {
       }
 
       if (allowedMembers.data.find(member => member.login === commentUser.login)) {
-        await COMMANDS[commandToRun](client);
+        const commandFn = COMMANDS[commandToRun];
+
+        await commandFn(client, commentBody);
+
+        // If it is a bot command, delete the comment.
```

Comment:
I'm actually iffy on this because we'll lose the trail of why it happened.

Reply:
It's a good point. Maybe it's better to remove this then.

Reply:
It would be better to hide it and mark as resolved, so it will be minimized but still be on in the thread, but Octokit doesn't have this yet. I will remove the delete part and when Octokit adds support we can review this.

Reply:
It's potentially in the GraphQL API, but I didn't check.